src/ua/ithillel/www/javapra/Mishnieva/homework/homawork5/ToList.java

```diff
@@ -0,0 +1,25 @@
+package ua.ithillel.www.javapra.Mishnieva.homework.homawork5;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ToList {
```

Comment:
this is incorrect name of the class, class name should represent see entity to the action